client/src/components/ListCard.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Edit from '../assets/edit.svg';
+import Delete from '../assets/delete.svg';
+import { action } from '@storybook/addon-actions';
+
+const ListCardContainer = styled.div`
+  display: flex;
+  background-color: ${(props) => props.theme.colors.primary};
+  width: 100%;
+  max-width: 500px;
+  height: 20vh;
+  max-height: 150px;
+  color: ${(props) => props.theme.colors.textprimary};
+  padding: 0px;
+  border: 1px solid black;
+`;
+
+const ListCardContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-basis: 80%;
+  padding-top: 10px;
+  padding-left: 10px;
+  font-size: 1.8rem;
+`;
+
+const ListSymbolContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-basis: 20%;
+  flex-direction: column;
+  justify-content: space-around;
+`;
+
+const ListSymbolImage = styled.img``;
+
+const ListExampleContent = ['ClientNameOne', 'ClientNameTwo'];
```

Comment:
```suggestion
const listExampleContent = ['ClientNameOne', 'ClientNameTwo'];
```

PascalCase is for React components

Reply:
Actually, this example should be in your story and not inside the component. 
Stories are great for test/example usage.

Reply:
I corrected the variable :) Maybe I will change the example location with the first refactor.